datafusion/physical-expr/src/aggregate/array_agg.rs

```diff
@@ -96,6 +109,139 @@ impl AggregateExpr for ArrayAgg {
     fn name(&self) -> &str {
         &self.name
     }
+
+    fn groups_accumulator_supported(&self) -> bool {
+        self.input_data_type.is_primitive() || self.input_data_type == DataType::Utf8
```

Comment:
Are we able to support LargeUtf8 and Binary as well? Given how similar they are to Utf8?